syftlib/build.gradle

```diff
@@ -29,6 +30,25 @@ task jacocoTestReport(type: JacocoReport, dependsOn: ['testDebugUnitTest', 'crea
     ])
 }
 
+
+dokka {
+    outputFormat = 'html'
+    outputDirectory = "$buildDir/html"
+    doLast {
+        ant.move file: "$outputDirectory",
+                todir: "$rootDir/docs"
+    }
+}
+
+task dokkaMarkdown(type: org.jetbrains.dokka.gradle.DokkaTask) {
+    outputFormat = 'markdown'
```

Comment:
Better use the gh-markdown as the format

Reply:
I can't see this format in Dokka.

Reply:
https://github.com/Kotlin/dokka#output_formats

Reply:
It couldn't be easier to find. Thanks.